### PR TITLE
react-router-dom

### PR DESCRIPTION
**Pull request by Caleb-Nieto**
### Feature / Bug Description
Adds react-router-dom dependency

### Solution
reac-router-dom dependency added and BrowserRoutes, Routes and Route component added on main.tsx file

### Notes
Any note

### Tickets
[WMA-18] (https://cdentalcaregroup.atlassian.net/browse/WMA-18)

### Screenshots / Screencasts
image

[WMA-18]: https://cdentalcaregroup.atlassian.net/browse/WMA-18?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by imadiaz — CHANGES_REQUESTED** [review]
please merge first this PR and then rebase from develop to have just the changes about this PR and not both PR's https://github.com/M2-App/web-m2-app/pull/4

File: src/App.tsx [hunk 1]
```diff
@@ -1,20 +1,26 @@
...
+          ))}
+        </Route>
+      </Routes>
     </ConfigProvider>
-  )
+  );
 }
 
-export default App
+export default App;
```
Comment (imadiaz) [inline src/App.tsx:26]: i think that you don't need ;

**Review by imadiaz — COMMENTED** [review]
can u add a video navigating between the options?

File: src/pages/routes/Routes.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
+import { ItemType } from "antd/es/menu/hooks/useItems";
+import { Page1, Page2, Page3 } from "../proofPages/proofPages";
+import {
+  UserOutlined,
+  VideoCameraOutlined,
+  UploadOutlined,
+} from "@ant-design/icons";
+import { MenuProps } from "antd";
+
+export class Route {
```
Comment (imadiaz) [inline src/pages/routes/Routes.tsx:10]: maybe we can extract this code in a class 

/routes/models/route.tsx
 and move just the class there

**Comment by Caleb-Nieto** [comment]
Of course!! 
 https://github.com/M2-App/web-m2-app/assets/133397335/d3e8d45a-8f3c-4c44-b0cd-a6e1ab8a4021

File: src/pages/proofPages/proofPages.tsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+export const Page1 = () => {
```
Comment (imadiaz) [inline src/pages/proofPages/proofPages.tsx:1]: Just to keep in mind once we add the pages, we can remove this mock pages

File: src/pages/routes/Routes.tsx (hunk 2 above)
Comment (Caleb-Nieto) [reply src/pages/routes/Routes.tsx:10]: Okay I like it!!

File: src/App.tsx (hunk 1 above)
Comment (Caleb-Nieto) [reply src/App.tsx:26]: Okay!!

File: src/pages/proofPages/proofPages.tsx (hunk 3 above)
Comment (Caleb-Nieto) [reply src/pages/proofPages/proofPages.tsx:1]: Yeah! I will remember

**Comment by Caleb-Nieto** [comment]
can you see the video?

**Review by imadiaz — APPROVED** [review]
lgtm

**Pull request merged**